src/layer/dataset.ts

```diff
@@ -209,6 +209,20 @@ export const DATASET_AWS_DEM: Dataset = {
   shWmsEvalsource: 'DEM',
   shProcessingApiDatasourceAbbreviation: 'DEM',
   datasetParametersType: null,
+  shServiceHostname: 'https://services.sentinel-hub.com/',
+  searchIndexUrl: null,
+  findDatesUTCUrl: null,
+  orbitTimeMinutes: null,
+  minDate: null,
+  maxDate: null,
+};
```

Comment:
Changing the existing dataset definitions is a backwards incompatible change. It is very likely that nobody uses this yet, but maybe it is better to avoid taking chances... I would suggest changing names so that this one (`DATASET_AWS_DEM`) is not used, and then just linking `DATASET_AWS_DEM` to the appropriate new name. Not a strong opinion though, as this probably wasn't used by anyone yet. Fftr.